packages/@ama-sdk/create/src/index.ts

```diff
@@ -26,20 +25,21 @@ if (argv._.length < 2) {
 }
 
 const sdkType = argv._[0];
-let [name, pck] = argv._[1].replace(/^@/, '').split('/');
 
 if (sdkType !== 'typescript') {
   console.error('Only the generation of "typescript" SDK is available');
   process.exit(-2);
 }
 
-if (!pck) {
-  console.warn(`The given package name, is not a scoped package. a default "@${defaultScope}" scope will be used`);
-  pck = name;
-  name = defaultScope;
+const fullPackage = argv._[1];
+const packageMatch = /^(?:@?([^@/]+)\/)?([^@/]+)$/.exec(fullPackage);
```

Comment:
Not sure to understand the first `?`, do you want to support path before the package name? In that case it is supported only for packages without scope :S
Maybe the simplest way would be to remove it:

```suggestion
const packageMatch = /^(?:@([^@/]+)\/)?([^@/]+)$/.exec(fullPackage);
```

Reply:
Indeed, it makes more sense to make it mandatory to write a correct package name, either with a scope (including the @ character) or without a scope. I had kept the previous behavior (which allowed to specify a scope without the @ character, and the create command was adding it), but I prefer what you suggest.